src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -160,7 +160,11 @@ export const RadioGroup: ChakraComponent<
             {showRequiredLabel && isRequired && <span> (required)</span>}
           </Box>
           <ChakraRadioGroup {...radioGroupProps}>
-            <Stack direction={[layout]} spacing={spacingProp}>
+            <Stack
+              className="radioGroupStack"
```

Comment:
I know we've done things like this before where a generic custom class is added to an element which then allows us to target the element in a situation where it is being used to compose another component. (Phew, longwinded!) Is this class name following the same naming convention that we have used for other class names like this?

Reply:
This discussion is related to [DSD-2038](https://newyorkpubliclibrary.atlassian.net/browse/DSD-2038). That ticket says, "Initial thought is to remove prefixes from classnames/ids that use them," which is interesting because my feeling here was to add some type of prefix to all of the classes in order to ensure that they were not too general and would not accidentally pick up unrelated styles from elsewhere.

What are you thinking?